xcpmd/src/parser.c

```diff
@@ -1888,11 +1889,27 @@ int parse_config_from_file(char * filename) {
     while (fgets(line, 1024, file)) {
 
         line_no++;
+        xcpmd_log(LOG_DEBUG, "Parsing line %d: %s", line_no, line);
 
         //Discard any line beginning with a comment character (#).
         if (line[0] == '#')
             continue;
 
+        //Discard any lines containing only whitespace.
+        whitespace_line = false;
+        ptr = line;
+        while (*ptr <= ' ') {
```

Comment:
The <=' ' should catch tabs, and a handful of other non-printable characters as well.